getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -51,34 +80,97 @@ data:
 --
 
 . Click *Save*.
-. Create a key or value Secret file named `rhdh-secrets` using the environment variables as keys from the previous code snippet by following the steps below:
+. Navigate to the *Helm* tab and select *Upgrade*.
+. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from existing Secrets*, input `github-secrets` as the value.
+. Click *Upgrade*.
+
+== Configuring GitHub integration
+
+The configuration of GitHub is required to enable the GitHub plugins in {product-short}.
+
+.Procedure
+
+. In the Red Hat OpenShift cluster, navigate to the main page of the GitHub organization where you want to create the OAuth application.
+. Click *Settings* -> *<> Developer Settings* -> *GitHub Apps* -> *New GitHub App*.
+. Enter the application name as `Developer Hub`.
+. Add the following as the *Homepage URL*:
 +
 --
-.. In Red Hat OpenShift, go to the *Secrets* tab and click *Create*.
-.. Enter *Secret name* as `rhdh-secrets`.
-.. Add environment variables as *Key* and *Value* and click *Create*.
+`https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/`
 --
 
-. To log in using your GitHub application, ensure that the *Callback URL* in your GitHub application is configured as follows:
+. Add the following URL as *Authorization callback URL*:
 +
 --
 `https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/api/auth/github/handler/frame`
+--
+
+. Unselect *Webhook URL* -> *Active*.
+. Under the *Where can this GitHub App be installed?* section, ensure that *Only on this account* is selected.
+. After creating the application, click *Generate a new client secret* and copy the generated client secret.
+. Click *Generate a private key* at the bottom of the page and download the generated file.
+. In OpenShift, click *ConfigMaps*.
+. Generate a key/value secret named 'github-secrets' using the provided environment variables as keys, and input the values you generated for your GitHub OAuth application:
++
+--
+.. In Red Hat OpenShift, go to the *Secrets* tab and click *Create*.
+.. Select *Key/value secret*.
+.. Enter *Secret name* as `github-secrets`.
+.. Add environment variables as *Key* and *Value* and click *Create*.
+
+.Environment variables
```

Comment:
If this is a result, consider adding the .Result heading to make the purpose of this section more clear to the user

Reply:
The table lists the environment variables that users need to add, along with their descriptions. These variables are generated depending on the user's environment.